File: ex-giturb/python/utils/urbs_seeder.py
import json
from models.urb import Urb
from utils.in_memory_datastore import urbs_data_store

# Seeder JSON string
urbs_json = """
            [
                { "id": 1, "name": "Basil" },
                { "id": 2, "name": "Rosemary" },
                { "id": 3, "name": "Thyme" },
                { "id": 4, "name": "Sage" },
                { "id": 5, "name": "Mint" },
                { "id": 6, "name": "Parsley" },
                { "id": 7, "name": "Cilantro" },
                { "id": 8, "name": "Dill" },
                { "id": 9, "name": "Oregano" },
                { "id": 10, "name": "Chives" }
            ]
            """

def seed_urbs():
    # Parse the JSON string
    urbs = json.loads(urbs_json)
    # Clear existing data
    urbs_data_store.clear()
    # Populate the data store
    for urb in urbs:
        urbs_data_store.append(urb)

File: ex-giturb/dotnet/giturb/Utils/UrbsSeeder.cs
public class UrbsSeeder
{
    public static List<Urb> SeedUrbs()
    {
        // Generate 10 urbs (urbs are the same as herbs) using real herbs names using json format
        var urbsJson = """
                [
                    { "Id": 1, "Name": "Basil" },
                    { "Id": 2, "Name": "Rosemary" },
                    { "Id": 3, "Name": "Thyme" },
                    { "Id": 4, "Name": "Sage" },
                    { "Id": 5, "Name": "Mint" },
                    { "Id": 6, "Name": "Parsley" },
                    { "Id": 7, "Name": "Cilantro" },
                    { "Id": 8, "Name": "Dill" },
                    { "Id": 9, "Name": "Oregano" },
                    { "Id": 10, "Name": "Chives" }
                ]
                """;

        return System.Text.Json.JsonSerializer.Deserialize<List<Urb>>(urbsJson);
    }
}
